system/jlib/jtrace.hpp

```diff
@@ -164,11 +164,23 @@ class jlib_decl OwnedSpanScope
 public:
     OwnedSpanScope() = default;
     OwnedSpanScope(ISpan * _ptr);
+    OwnedSpanScope(const OwnedSpanScope& rhs) = delete;
+    OwnedSpanScope(OwnedSpanScope&& rhs) = default;
     ~OwnedSpanScope();
 
     inline ISpan * operator -> () const         { return span; }
     inline operator ISpan *() const             { return span; }
 
+    inline OwnedSpanScope& operator=(ISpan * ptr) = delete;
+    inline OwnedSpanScope& operator=(const OwnedSpanScope& rhs) = delete;
+    inline OwnedSpanScope& operator=(OwnedSpanScope&& rhs)
```

Comment:
Given thoughts on HPCC-32982 I think this should probably be deleted.  I'm not sure it makes sense because of the additional semantics of setting the thread active span - the user should only use set/setown.